pkg/scalers/aws_cloudwatch_test.go

```diff
@@ -436,7 +452,7 @@ func TestAWSCloudwatchScalerGetMetrics(t *testing.T) {
 		case testAWSCloudwatchErrorMetric:
 			assert.Error(t, err, "expect error because of cloudwatch api error")
 		case testAWSCloudwatchNoValueMetric:
-			assert.Error(t, err, "expect error because of no data return from cloudwatch")
+			assert.NoError(t,err, "dont expect error when returning empty metric list from cloudwatch")
```

Comment:
`assert.NoError(t,err, "dont expect error when returning empty metric list from cloudwatch")` should be
`assert.NoError(t, err, "dont expect error when returning empty metric list from cloudwatch")`.
Otherwise the linter fails